src/uu/ls/src/ls.rs

```diff
@@ -45,19 +45,12 @@ use unicode_width::UnicodeWidthStr;
 #[cfg(unix)]
 use uucore::libc::{S_IXGRP, S_IXOTH, S_IXUSR};
 
-static ABOUT: &str = "
- By default, ls will list the files and contents of any directories on
- the command line, expect that it will ignore files and directories
- whose names start with '.'
-";
-static AFTER_HELP: &str = "The TIME_STYLE argument can be full-iso, long-iso, iso.
-Also the TIME_STYLE environment variable sets the default style to use.";
-
```

Comment:
Why are you moving these?

The current form follows the pattern of other utils, so it should remain unless you have some reason to move them.

Reply:
In addition to @rivy's comment, we can consider moving those constants under our `mod options` but yeah please don't break the consistency for now. 

```rust
pub mod options {
    pub mod help {
        pub static HELP: &str = "help";
    }
}